stubs/jsonschema/jsonschema/protocols.pyi

```diff
@@ -0,0 +1,23 @@
+from typing import Any, ClassVar, Iterator, Optional, Union
+from typing_extensions import Protocol
```

Comment:
We (incorrectly) include `Protocol` in `typing.pyi` unconditionally, which is why we import it directly from `typing` in typeshed.

Reply:
I updated to include `Protocol` in the `from typing import ...` blob.

I can't comment on the (in)correctness of doing it one way or another, but I can do a PR to make all other stubs consistent about this, if that sounds valuable?
I saw at least one example of each import for `Protocol`.

Reply:
Ideally, `Protocol` would be guarded behind a version check, but changing it now would cause too much disruption. (And is obsolete in a few years when we drop support for Python 3.7.) So there's no reason not to import directly from typing. We certainly wouldn't mind a PR that makes this consistent across typeshed, but don't feel obligated to submit one.

Reply:
I went ahead and put together a PR. It wasn't much trouble, and it gets this out of the way for when 3.7 is removed. Hopefully it pays dividends by making this unambiguous for future contributors!